presto-main/src/main/java/com/facebook/presto/sql/planner/CanonicalPlanFragment.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
This class would be very similar to class `CanonicalPlan`. How we can abstract them in a way to move them closer? Let's chat in person.